interfaces/page-info.ts

```diff
@@ -29,6 +29,12 @@ export type ChildPageInfo = {
   title: string;
 };
 
+// @TODO: Rename this
```

Comment:
This might get removed as the other templates get added. I will rename it if it still needs to exist.